Documentation/Signoffs/PLCSubsystem.md

```diff
@@ -0,0 +1,74 @@
+|PLC|6|PLC shall be able to input data from at least 4 Arduino Nano 33 IoT and output to 1 HMI|
+
+## **Buildable Schematic:**
+
+![PLC Design](https://github.com/RealityHertz/Greenhouse-Project/blob/main/Documentation/Images/CADPLCSubsystemDrawing.jpg)
+
+*Figure 1. PLC Subsystem Design*
+
+![PLC Subsystem](https://github.com/RealityHertz/Greenhouse-Project/blob/main/Documentation/Images/PLC%202D%20Model.png)
+
+*Figure 2. PLC 2D Model*
+
+![PLC Power Supply](https://github.com/RealityHertz/Greenhouse-Project/blob/main/Documentation/Images/PLCPowerSupply2DCAD.PNG)
+
+*Figure 3. PLC Power Supply 2D Model*
+
+## **Analysis:**
+
+**PLC**
+  1. The Automation Direct C2-03CPU has a scan time of < 1 ms and a contact execution of < 0.2 µs. This is more than sufficient to keep track of the sensor's data being sent every 5 minutes.
+  2. The Automation Direct C2-03CPU functions on a Bluetooth frequency of 2.4 GHz. This is more than sufficient to handle the connection between the PLC and the various subsystem sensors.
+  3. The Automation Direct C2-03CPU has a Bluetooth linespeed of 260 kbps. This is fast enough to process the data transfer between plc and sensors within the 5-minute cycles.
+  4. The CLICK C0-01AC will supply the PLC with a constant output range of 23-35V with an 85-264 input voltage. It will also output a current of 1.3A with an over-current protection system with automatic recovery at 1.6A. This power supply also has an overvoltage protection system of 27.6V clamped by a zener diode. This power supply holds the same temperature and humidity requirements as the PLC as well. [4]
+  5. The ambient humidity specifications in the PLC Datasheet show that when operating and non-condensing the relative humidity the PLC can handle is 30% to 95%. [2] This falls within our range for the greenhouse, with 80% being optimal for plant growth. [3] The PLC will also be able to monitor the humidity through communication with the Temperature/Humidity Subsystem's sensor to alert if the humidity is falling or rising to an unsafe range.
+  6. We will utilize the Greenhouse Wifi along with Arduino onboard IoT (Internet of Things) technology and MQTT (Message Queuing Telemetry Transport) protocol to have communication between all Arduino devices and the PLC. Through the CLICK CPU v3.0 Application and an additional RP-SMA antenna plug directly connected to the PLC, we can establish connections with up to 8 additional IoT devices. The PLC, by default, supports one HMI (Human-Machine Interface), but it can be configured in the CLICK CPU to have more interfaces as needed. The primary focus will be on using IoT capabilities for communication with multiple Arduino Nano 33 IoT devices. This approach ensures reliable communication between the PLC and the Arduino devices within the greenhouse environment. [1]
```

Comment:
This has not been discussed. You have not designed such a system. You'll have to prove all of this will work. It is insufficient to claim it. It must be designed and analyzed here.